tests/integ/modin/groupby/test_groupby_negative.py

```diff
@@ -536,7 +536,10 @@ def test_groupby_agg_invalid_numeric_only(
     # treated as True. This behavior is confusing to customer, in Snowpark pandas, we do an
     # explicit type check, an errors it out if an invalid value is given.
     with pytest.raises(
-        ValueError, match=re.escape('For argument "numeric_only" expected type bool')
+        ValueError,
+        match=re.escape(
+            "GroupBy aggregations like 'sum' take a 'numeric_only' argument that needs to be a bool, but a str value was passed in."
```

Comment:
I think you need to parametrize by `"numeric_only,type_name"` since int and string have different names.

Reply:
Good catch - but instead of adding extra parameter, I just updated the error message to be an f-string and use {type(numeric_only).__name__}, same as where the error message is thrown.